kinto/config/kinto.tpl

```diff
@@ -65,22 +119,67 @@ multiauth.policies = basicauth
 # Set user 'account:admin' as the administrator.
 # kinto.account_write_principals = account:admin
 # kinto.account_read_principals = account:admin
+#
+# Kinto-portier authentication
+# https://github.com/Kinto/kinto-portier
+# multiauth.policies = portier
+# multiauth.policy.portier.use = kinto_portier.authentication.PortierOAuthAuthenticationPolicy
+# kinto.portier.broker_url = https://broker.portier.io
+# kinto.portier.webapp.authorized_domains = *.github.io
+# kinto.portier.cache_ttl_seconds = 300
+# kinto.portier.state.ttl_seconds = 3600
 
+# Notifications
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#notifications
 #
-# Firefox Accounts configuration.
-#   These are working FxA credentials for localhost:8888
-# kinto.includes  = kinto_fxa
-# fxa-oauth.client_id = 61c3f791f740c19a
-# fxa-oauth.client_secret = b13739d8a905315314b09fb7b947aaeb62b47c6a4a5efb00c378fdecacd1e95e
-# fxa-oauth.oauth_uri = https://oauth-stable.dev.lcip.org/v1
-# fxa-oauth.requested_scope = profile kinto
-# fxa-oauth.required_scope = kinto
-# fxa-oauth.relier.enabled = true
-# fxa-oauth.webapp.authorized_domains = *
+# Configuration example:
+# kinto.event_listeners = redis
+# kinto.event_listeners.redis.use = kinto_redis.listeners
+# kinto.event_listeners.redis.url = redis://localhost:6379/0
+# kinto.event_listeners.redis.pool_size = 5
+# kinto.event_listeners.redis.listname = queue
+# kinto.event_listeners.redis.actions = create
+# kinto.event_listeners.redis.resources = bucket collection
 
+# Cross Origin Requests
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#cross-origin-requests-cors
 #
-# Client cache headers
+# kinto.cors_origins = *
+
+# Backoff indicators/end of service
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#backoff-indicators
+# https://kinto.readthedocs.io/en/latest/api/1.x/backoff.html#id1
+#
+# kinto.backoff =
+# kinto.retry_after_seconds = 3
+# kinto.eos =
+# kinto.eos_message =
+# kinto.eos_url =
+
+# Project information
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#project-information
 #
+# kinto.version_json_path = ./version.json
+# kinto.error_info_link = https://github.com/kinto/kinto/issues/
+# kinto.project_docs = https://kinto.readthedocs.io
+# kinto.project_version = ''
+# kinto.version_prefix_redirect_enabled = True
+
+# Enabling or disabling endpoints
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#enabling-or-disabling-endpoints
+# Disable article collection DELETE endpoint
+# Where: - endpoint_type is either collection or record; - resource_name is the name of the resource (by default, Kinto uses # the name of the class); - method is the http method (in lower case):
+# Ex:
+# kinto.collection_article_delete_enabled = false
+# kinto.record_mushroom_patch_enabled = false
```

Comment:
I realize this won't work in Kinto itself. Indeed, the resources names in Kinto are only `bucket`, `group`, `collection`, `record`.

I know this is very confusing ( see #710 ) because the endpoint types are also named `collection` (plural) and `record` (instance) :( :(
So the examples are like `kinto.collection_bucket_delete_enabled = false` or `kinto.record_record_delete_enabled = false`.
It is so confusing that maybe we could even skip this section.... please decide :) your call!